scripts/data_transfer/copy_test_gvcfs.py

```diff
@@ -5,25 +5,47 @@
 
 output = os.getenv('OUTPUT')
 assert output and output.startswith('gs://cpg-tob-wgs-test/')
-
-SUBSET_RE = 'TOB15[2-3]'
 QC_METADATA_FILE_NAME = 'R_210315_BINKAN1_1K1KDNA_M001.csv'
 
-subprocess.run(
-    [
-        'gsutil',
-        'cp',
-        f'gs://cpg-tob-wgs-main/gvcf/batch0/{SUBSET_RE}?.g.vcf.gz*',
-        'gs://cpg-tob-wgs-test/gvcf/batch0/',
-    ],
-    check=False,
-)
+SAMPLES = [
+    'TOB1520',
+    'TOB1521',
+    'TOB1522',
+    'TOB1523',
+    'TOB1524',
+    'TOB1525',
+    'TOB1526',
+    'TOB1527',
+    'TOB1528',
+    'TOB1529',
+    'TOB1530',
+    'TOB1531',
+    'TOB1532',
+    'TOB1533',
+    'TOB1534',
+    'TOB1535',
+    'TOB1536',
+    'TOB1537',
+    'TOB1538',
+    'TOB1640',
+]
+
+for sn in SAMPLES:
+    subprocess.run(
```

Comment:
Note that you can now also run shell scripts directly, which might be a bit more convenient for tasks like these.

Doesn't matter much here, but if you use `gsutil -m cp`, you can copy in parallel, whereas this `for` loop will do things sequentially.

Reply:
Nice, I knew about `gsutil -m cp` but haven't really used it much. Should be definitely useful for other scenarios.